interface_frontend.py

```diff
@@ -132,6 +131,10 @@ def on_draw():
 def on_text(text):
     label.text = text
     print(text)
+    if text == 'p' and interface_state.power_down == False:
+        interface_state.power_down = True
+    elif text == 'p':
+        interface_state.power_down = False
```

Comment:
Nad podobnou logikou jsem se zamýšlel několikrát (v různých projektech). Přijde mi že přehlednější je napsat:

```python
if text == 'p':
    if interface_state.power_down == False:
        interface_state.power_down = True
    else:
        interface_state.power_down = False
```

... aby bylo jasné hned z toho prvního `if`, že tenhle celý kousek kódu se dělá jen pro tlačítko `p`.

Vypadá to takhle přehledněji? Dá se to zpřehlednit víc?